### DISCUSSION TITLE
Letter Spacing Not Working on TextInput

### DISCUSSION THREAD

**Discussion by thr3a** [mantinedev/mantine#6058] [Q&A]
Hello,

I'm encountering an issue with the `TextInput` component where the `letterSpacing` CSS property doesn't seem to be taking effect. I'm trying to increase the spacing between characters, but I'm unsure of the correct approach. Here's what I've tried so far:

```tsx
<TextInput value={'aaaa'} />
<TextInput style={{ letterSpacing: '10px' }} value={'aaaa'} />
<TextInput lts={'10px'} value={'aaaa'} />
<TextInput lts={10} value={'aaaa'} />
```

None of these attempts have resulted in increased letter spacing. Could someone guide me on how to properly apply letterSpacing to a TextInput component?

Thank you.

![スクリーンショット 2024-04-10 13 02 15](https://github.com/mantinedev/mantine/assets/6120301/a57da66b-017d-4426-9a1e-6200b5ae5ed3)

**Comment by rtivital** [comment]
You need to apply styles to the input element with Styles API – https://mantine.dev/styles/styles-api/

**Comment by thr3a** [comment]
The code below worked. thank you!

```tsx
<TextInput styles={{ input: { letterSpacing: '10px' } }} value={'aaaa'} />
```
